src/main/scala/dpla/ingestion3/OaiRecordHarvesterMain.scala

```diff
@@ -121,6 +118,11 @@ object OaiRecordHarvesterMain {
       case (Some(sets), Some(blacklist)) => {
         baseOptions + ("sets" -> (sets.split(",") filter (!blacklist.split(",").contains(_))).mkString(","))
       }
+      // No set list passed in but a blacklist was passed in
+      // TODO: Should this error out or should it try to call ListSets to generate a set list?
```

Comment:
I'm confused. If you have a set list locally, you wouldn't need a blacklist because you could just remove the items manually.